README.md

```diff
@@ -0,0 +1,6 @@
+# OptionsResolver
+[![Build Status](https://github.com/bim-g/wepesi-router/actions/workflows/php.yml/badge.svg)](https://github.com/bim-g/OptionsResolver/actions) [![Source](http://img.shields.io/badge/source-bimg/router-blue.svg?style=flat-square)](https://github.com/bim-g/OptionsResolver) [![Version](https://img.shields.io/packagist/v/wepesi/optionsresolver.svg?style=flat-square)](https://packagist.org/packages/wepesi/optionsresolver) [![Downloads](https://img.shields.io/packagist/dt/wepesi/optionsresolver.svg?style=flat-square)](https://packagist.org/packages/wepesi/optionsresolver/stats) [![License](https://img.shields.io/packagist/l/wepesi/optionsresolver.svg?style=flat-square)](https://github.com/bim-g/OptionsResolver/blob/master/LICENSE)
```

Comment:
This element should be update coresponding to the project and make sure they are working fine.